docs/further.md

```diff
@@ -0,0 +1,33 @@
+# Further Reading
+
+## Presentations
+- Aqua Security YouTube Channel
+  - [Trivy - container image scanning][intro]
+  - [Using Trivy in client server mode][server]
+  - [Tweaking Trivy output to fit your workflow][tweaking]
+  - [How does a vulnerability scanner identify packages?][identify]
```

Comment:
I run preview by using similar goal to https://github.com/aquasecurity/starboard/blob/main/Makefile#L126

In this markdown flavour you need 2 more space characters to have proper nesting of list: 
```suggestion
    - [Trivy - container image scanning][intro]
    - [Using Trivy in client server mode][server]
    - [Tweaking Trivy output to fit your workflow][tweaking]
    - [How does a vulnerability scanner identify packages?][identify]
```

Reply:
Thanks!